docs/interactions.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
+
+.. currentmodule:: nextcord
+
+.. _client_index:
+   
+Interaction Commands
+======================
+
+As discord have added interaction commands a feature with alot of possiblities and ideas we at nextcord have decided to add them to our fleet of features,
```

Comment:
To make this document easier to read and more formal, I suggest these change's. 

"Since Discord has added interaction commands, a feature with many possibilities, we at Nextcord have decided to add them to our fleet of features."